.github/workflows/checks.yaml

```diff
@@ -49,6 +49,9 @@ jobs:
       - name: Run integration tests
         shell: bash
         run: pytest tests/test_integration
+      - name: Run unit tests
+        shell: bash
+        run: pytest tests/unitTest/pro_tes/middleware
```

Comment:
If you name your directories and files `*_test` or `test_*`, `pytest` will autodiscover all tests and you don't need to specify them on the command line. See here for details: https://docs.pytest.org/en/7.2.x/explanation/goodpractices.html

In any case, Python naming conventions dictate snake case, so we should stick to that.

Reply:
Actually, there are some tests that are pretty old and need to be rewritten (which are at `tests/unitTest/pro_tes/endpoints`) otherwise they will always fail, that's why excluded them by mentioning only the required tests to run ( i.e. inside `tests/unitTest/pro_tes/middleware`).
What I can do now is rename the directories as you mentioned, now, but once all the test inside the unitTest directory is running fine, we can then change the command line to  `pytest tests/unittest/`

Reply:
Oh, I didn't know we _had_ unit tests :) Please fix them ASAP and rename the packages/modules according to Python/`pytests` naming conventions and update the `pytest` call as you say :pray: